tools/kbump/cmd/kbump/main.go

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Can we add some functional testing here to ensure that we get the output that we would expect? Some basic Gingko tests should suffice here I would think.

Reply:
Test cases added for both provisioner and node template